constants.js

```diff
@@ -341,6 +384,18 @@ export const chainInfos = {
         tmVersion: COMETBFT_VERSIONS.comet37,
         pstakeConfig: pstakeHostValsetConfigs.osmosis
     },
+    dydx: {
+        rpc: "https://r-dydx-archive-sub--atnqqmfffe9qgz02sjhwcnk35nu4vil6.gw.notionalapi.com:443",
+        // rpc: "https://dydx-mainnet-full-rpc.public.blastapi.io:443",
+        // rpc: "https://dydx-rpc.kingnodes.com:443",
+        // rpc: "https://dydx-dao-rpc.polkachu.com:443",
+        chainID: "dydx-mainnet-1",
+        prefix: "dydx",
+        feeDenom: "adydx",
+        gasPrice: GasPrice.fromString("0.005adydx"),
```

Comment:
is this ok? , anyway not required as of now..

Reply:
exactly, just put it as a placeholder